channels/routing.py

```diff
@@ -52,6 +53,70 @@ async def __call__(self, scope, receive, send):
             )
 
 
+def _parse_resolver(child_url_pattern, parent_resolver, parent_regex, routes):
+    """Parse resolver (returned by `include`) recurrsively
+
+    Parameters
+    ----------
+    child_url_pattern : URLResolver |
```

Comment:
Why the trailing `|`?

Reply:
Ah, I forgot to append to Any.
This function receives the url_pattern returned by `path` or `re_path` and `include` (URLResolver)